### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by ozh**
Onto 1.9

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ YOURLS Changelog
 _This file lists the main changes through all versions of YOURLS.  
 For a much more detailed list, simply refer to [commit messages](https://github.com/YOURLS/YOURLS/commits/master)._
 
+1.8.3
+---
+- improved: the API plugin -- more plugin functions (#3281), a sandbox and a plugin uninstall procedure (#3282)
+- improved: inline documentation, [online documentation](https://doc.yourls.org/) and unit tests
+- improved: concurrency during mass shortening (#3233)
+- improved: minor security fixes - sanitize step name during upgrade (#3055),
+    nonce on the logout link (#3264), salt cookie with newer hash (#3278),
```
Comment (dgw) [inline CHANGELOG.md:13]:
```suggestion
    nonce on the logout link (#3264), salt cookie with newer hash (#3278)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ YOURLS Changelog
 _This file lists the main changes through all versions of YOURLS.  
 For a much more detailed list, simply refer to [commit messages](https://github.com/YOURLS/YOURLS/commits/master)._
 
+1.8.3
+---
+- improved: the API plugin -- more plugin functions (#3281), a sandbox and a plugin uninstall procedure (#3282)
+- improved: inline documentation, [online documentation](https://doc.yourls.org/) and unit tests
```
Comment (LeoColomb) [inline CHANGELOG.md:10]:
```suggestion
- improved: inline documentation, [online documentation](https://docs.yourls.org/) and unit tests
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,19 @@ YOURLS Changelog
...
+- improved: concurrency during mass shortening (#3233)
+- improved: minor security fixes - sanitize step name during upgrade (#3055),
+    nonce on the logout link (#3264), salt cookie with newer hash (#3278),
+- improved: Remove ozh/phpass library and use password_* functions (#3232)
```
Comment (LeoColomb) [inline CHANGELOG.md:14]:
```suggestion
- improved: Remove ozh/phpass library and use native PHP password_* functions (#3232)
```

**Review by LeoColomb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,19 @@ YOURLS Changelog
...
+- improved: third party binaries will now be automatically updated in the repository
```
Comment (LeoColomb) [inline CHANGELOG.md:15]: This is really nice for us, but can lead user to misinterpretation: third party binaries will _not_ be automatically updated from user POV.

**Review by LeoColomb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (ozh) [reply CHANGELOG.md:15]: True. I'll mention this as a "nice to have for us" in the blog post, and point to [ozh/yourls-geoip-update](https://github.com/ozh/yourls-geoip-update) as a user friendly way to update anytime

**Review by LeoColomb — APPROVED** [review]

**Pull request merged**